python/hsfs/core/online_store_sql_engine.py

```diff
@@ -19,8 +19,10 @@
 import json
 import logging
 import re
+import sys
 from typing import Any, Dict, List, Optional, Set, Tuple, Union
 
+import nest_asyncio
```

Comment:
Can you make this an optional import by importing only if is_runtime_notebook?

Reply:
done